plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/listeners/ServiceManagerEnvironment.java

```diff
@@ -93,7 +97,9 @@ public static ServiceManagerEnvironment loadServiceManagerEnvironment(IServer se
 	 */
 	public static ServiceManagerEnvironment loadServiceManagerEnvironment(IServer server) {
 		
-		String[] args = new String[]{CDKConstants.VAGRANT_CMD_SERVICE_MANAGER, CDKConstants.VAGRANT_CMD_SERVICE_MANAGER_ARG_ENV, "docker"};
```

Comment:
@robstryker I'm pretty sure you told me old versions of the service-manager plugin wouldn't support just `vagrant service-manager env`